lib/shared/sdk/usbboot/index.js

```diff
@@ -251,6 +251,21 @@ class USBBootAdapter extends EventEmitter {
     usb.listDevices().filter(isUsbBootCapableUSBDevice).map((device) => {
     /* eslint-enable lodash/prefer-lodash-method */
 
+      if (!device.accessible) {
+        return {
+          device: `${usbIdToString(device.deviceDescriptor.idVendor)}:${usbIdToString(device.deviceDescriptor.idProduct)}`,
+          displayName: 'Missing drivers',
+          description: 'Compute Module',
```

Comment:
I guess one refinement here would be to display separate labels for `Compute Module 1` and `Compute Module 3` based on the `.idProduct` (since this will be at the stage from before usbboot is run, so the PIDs are still distinct)

Reply:
Done!